WordPress/src/main/java/org/wordpress/android/ui/people/PersonDetailFragment.java

```diff
@@ -78,7 +136,7 @@ public void onResume() {
     public void refreshPersonDetails() {
         if (!isAdded()) return;
 
-        Person person = PeopleTable.getPerson(mPersonID, mLocalTableBlogID);
+        Person person = getCurrentPerson();
```

Comment:
Nitpicking: Let's lose the "Current" from the function name as it has some connotations (like, a globally selected "current" user) beyond the actual usage here. What do you think about "loadPerson()"? Cheers!